ClipboardAPI/clipboard-change-event-explainer.md

```diff
@@ -76,92 +71,102 @@ Web based editors like Word Online, Excel Online, Google Sheets, etc. may suppor
 #### 2.2.3 Multiple paste options in Google sheets
 ![](img/google-sheets.png)
 
+### 2.2 Scenario: Sync clipboard with a remote desktop
+When a user copies text or an image on their local machine, a web-based remote desktop application can detect that clipboard contents have changed by listening for the `clipboardchange` event. Upon detecting the change (which happens when 'clipboardchange' event is triggered on the web app when the page regains focus), the application can re-read the clipboard and send the updated clipboard content to the remote desktop environment.
+
+![](img/sync-clipboard-scenario.png)
+
+
 ## 3. Motivation - Alternative to inefficient polling of clipboard
 Today, a web-app can monitor the system clipboard by polling and reading the clipboard through async clipboard API at regular intervals.
 However, polling is not efficient and this feature aims to introduce an efficient way of notifying web apps when clipboard changes.
 Additionally we must ensure that we monitor the clipboard only when absolutely required, that is, there is at least one document having required permissions and is listening to the clipboard change event. This will be described in design details.
 
-## 4. Example javascript code for detecting clipboard changes:
+## 4. Proposed Approach
+
+### 4.1 Example javascript code for detecting clipboard changes:
```

Comment:
We should list the IDL changes we are proposed along side the example. As is, it's hard to grasp the full extent of the proposal.